src/main/java/org/klodnicki/entity/BookInfo.java

```diff
@@ -79,12 +74,12 @@ public int getCopiesNumber() {
         return copiesNumber;
     }
 
-    public void addAccount(Account account) {
-        accounts.add(account);
+    public void addLendingInformationAboutBooksList(LendingInformation lendingInformation) {
```

Comment:
Also naming: Could you find a better more precise name of what does it do? Right now it sounds like it sets the value.

Reply:
allright I changed that into addLendingInfoToList() I hope that works fine